src/NuGet.Core/NuGet.Build.Tasks.Pack/NuGet.Build.Tasks.Pack.csproj

```diff
@@ -94,9 +95,19 @@
     </ItemGroup>
     <PropertyGroup>
       <PathToMergedNuGetPack>$(ILMergeResultDir)\NuGet.Build.Tasks.Pack.dll</PathToMergedNuGetPack>
-      <IlmergeCommand>$(ILMergeExePath) /out:$(PathToMergedNuGetPack) $(ILMergePlatformArg)  /lib:$(OutputPath)$(RuntimeIdentifier)\publish /internalize /xmldocs /log:$(ILMergeResultDir)IlMergeLog.txt</IlmergeCommand>
-      <IlmergeCommand Condition="Exists($(MS_PFX_PATH))">$(IlmergeCommand) /delaysign /keyfile:$(MS_PFX_PATH)</IlmergeCommand>
-      <IlmergeCommand>$(IlmergeCommand) $(OutputPath)NuGet.Build.Tasks.Pack.dll @(BuildArtifacts, ' ')</IlmergeCommand>
+      <IlmergeCommand>$(ILMergeExePath) ^
+        /out:$(PathToMergedNuGetPack) ^
+        $(ILMergePlatformArg) ^
+        /lib:$(OutputPath)publish ^
+        /internalize ^
+        /xmldocs ^
+        /log:$(ILMergeResultDir)IlMergeLog.txt</IlmergeCommand>
+      <IlmergeCommand Condition="Exists($(MS_PFX_PATH))">$(IlmergeCommand) ^
+        /delaysign ^
+        /keyfile:$(MS_PFX_PATH)</IlmergeCommand>
+      <IlmergeCommand>$(IlmergeCommand) ^
+        $(OutputPath)NuGet.Build.Tasks.Pack.dll ^
+        @(BuildArtifacts, ' ^&#x0A;')</IlmergeCommand>
```

Comment:
Did `' '` cause problems? I'm, asking because this is unreadable.

Reply:
No. I added `' ^^#x0A;'` to add a new line in the command for the MSBuild Exec task to improve readability in the logs

Reply:
What's the from and to in the logs?

Reply:
Original command --edited-- (From):

```
C:\repos\NuGet.Client\packages\ilmerge\3.0.21\tools\net452\ILMerge.exe /out:C:\repos\NuGet.Client\artifacts\NuGet.Build.Tasks.Pack\16.0\bin\Debug\net472\ilmerge\NuGet.Build.Tasks.Pack.dll /targetplatform:v4,"C:\Program Files (x86)\Reference Assemblies\Microsoft\Framework\.NETFramework\v4.7.2" /lib:C:\repos\NuGet.Client\artifacts\NuGet.Build.Tasks.Pack\16.0\bin\Debug\net472\publish /internalize /xmldocs /log:C:\repos\NuGet.Client\artifacts\NuGet.Build.Tasks.Pack\16.0\bin\Debug\net472\ilmergeIlMergeLog.txt /delaysign /keyfile:C:\repos\NuGet.Client\build\..\keys\35MSSharedLib1024.snk C:\repos\NuGet.Client\artifacts\NuGet.Build.Tasks.Pack\16.0\bin\Debug\net472\NuGet.Build.Tasks.Pack.dll C:\repos\NuGet.Client\artifacts\NuGet.Build.Tasks.Pack\16.0\bin\Debug\net472\Newtonsoft.Json.dll C:\repos\NuGet.Client\artifacts\NuGet.Build.Tasks.Pack\16.0\bin\Debug\net472\NuGet.Commands.dll C:\repos\NuGet.Client\artifacts\NuGet.Build.Tasks.Pack\16.0\bin\Debug\net472\NuGet.Common.dll C:\repos\NuGet.Client\artifacts\NuGet.Build.Tasks.Pack\16.0\bin\Debug\net472\NuGet.Configuration.dll C:\repos\NuGet.Client\artifacts\NuGet.Build.Tasks.Pack\16.0\bin\Debug\net472\NuGet.Credentials.dll C:\repos\NuGet.Client\artifacts\NuGet.Build.Tasks.Pack\16.0\bin\Debug\net472\NuGet.DependencyResolver.Core.dll C:\repos\NuGet.Client\artifacts\NuGet.Build.Tasks.Pack\16.0\bin\Debug\net472\NuGet.Frameworks.dll C:\repos\NuGet.Client\artifacts\NuGet.Build.Tasks.Pack\16.0\bin\Debug\net472\NuGet.LibraryModel.dll C:\repos\NuGet.Client\artifacts\NuGet.Build.Tasks.Pack\16.0\bin\Debug\net472\NuGet.Packaging.dll C:\repos\NuGet.Client\artifacts\NuGet.Build.Tasks.Pack\16.0\bin\Debug\net472\NuGet.ProjectModel.dll C:\repos\NuGet.Client\artifacts\NuGet.Build.Tasks.Pack\16.0\bin\Debug\net472\NuGet.Protocol.dll C:\repos\NuGet.Client\artifacts\NuGet.Build.Tasks.Pack\16.0\bin\Debug\net472\NuGet.Versioning.dll
```

From MSBuild Structured Log Viewer (To):
```
CommandLineArguments = C:\repos\NuGet.Client\packages\ilmerge\3.0.21\tools\net452\ILMerge.exe ^
        /out:C:\repos\NuGet.Client\artifacts\NuGet.Build.Tasks.Pack\16.0\bin\Debug\net472\ilmerge\NuGet.Build.Tasks.Pack.dll ^
        /targetplatform:v4,"C:\Program Files (x86)\Reference Assemblies\Microsoft\Framework\.NETFramework\v4.7.2" ^
        /lib:C:\repos\NuGet.Client\artifacts\NuGet.Build.Tasks.Pack\16.0\bin\Debug\net472\publish ^
        /internalize ^
        /xmldocs ^
        /log:C:\repos\NuGet.Client\artifacts\NuGet.Build.Tasks.Pack\16.0\bin\Debug\net472\ilmergeIlMergeLog.txt ^
        /delaysign ^
        /keyfile:C:\repos\NuGet.Client\build\..\keys\35MSSharedLib1024.snk ^
        C:\repos\NuGet.Client\artifacts\NuGet.Build.Tasks.Pack\16.0\bin\Debug\net472\NuGet.Build.Tasks.Pack.dll ^
        C:\repos\NuGet.Client\artifacts\NuGet.Build.Tasks.Pack\16.0\bin\Debug\net472\Newtonsoft.Json.dll ^
C:\repos\NuGet.Client\artifacts\NuGet.Build.Tasks.Pack\16.0\bin\Debug\net472\NuGet.Commands.dll ^
C:\repos\NuGet.Client\artifacts\NuGet.Build.Tasks.Pack\16.0\bin\Debug\net472\NuGet.Common.dll ^
C:\repos\NuGet.Client\artifacts\NuGet.Build.Tasks.Pack\16.0\bin\Debug\net472\NuGet.Configuration.dll ^
C:\repos\NuGet.Client\artifacts\NuGet.Build.Tasks.Pack\16.0\bin\Debug\net472\NuGet.Credentials.dll ^
C:\repos\NuGet.Client\artifacts\NuGet.Build.Tasks.Pack\16.0\bin\Debug\net472\NuGet.DependencyResolver.Core.dll ^
C:\repos\NuGet.Client\artifacts\NuGet.Build.Tasks.Pack\16.0\bin\Debug\net472\NuGet.Frameworks.dll ^
C:\repos\NuGet.Client\artifacts\NuGet.Build.Tasks.Pack\16.0\bin\Debug\net472\NuGet.LibraryModel.dll ^
C:\repos\NuGet.Client\artifacts\NuGet.Build.Tasks.Pack\16.0\bin\Debug\net472\NuGet.Packaging.dll ^
C:\repos\NuGet.Client\artifacts\NuGet.Build.Tasks.Pack\16.0\bin\Debug\net472\NuGet.ProjectModel.dll ^
C:\repos\NuGet.Client\artifacts\NuGet.Build.Tasks.Pack\16.0\bin\Debug\net472\NuGet.Protocol.dll ^
C:\repos\NuGet.Client\artifacts\NuGet.Build.Tasks.Pack\16.0\bin\Debug\net472\NuGet.Versioning.dll
```

Reply:
I'm not seeing the impact of the weird char in this case. 

Anyway, don't want to block on this, my only point is because that's a very common MSBuild pattern `@(BuildArtifacts, ' ')`, not something that the NuGet team does, and this causes a pause for me reading it doesn't seem to be adding much.

Reply:
Ok, reverted back